modules/sdk/project-templates/project-templates-service-builder/src/main/resources/archetype-resources/__artifactId__-api/bnd.bnd

```diff
@@ -6,4 +6,5 @@ Export-Package:\
 	${package}.model,\
 	${package}.service,\
 	${package}.service.persistence
+-check: EXPORTS
```

Comment:
Hey @Camork  Can you add a unit test that will create a project, change the package in the service.xml and run the 'buildService' task and then try to build the jar and assert that the build fails with the export error?